simple_salesforce/api.py

```diff
@@ -411,6 +411,12 @@ def __init__(self, object_name, session_id, sf_instance, sf_version='27.0', prox
         self.request = requests.Session()
         self.request.proxies = proxies
 
+        self.headers = headers if headers else {
```

Comment:
Just updated code to perform a dict update on original header definition w/ passed in headers. Let me know your thoughts - thanks!